MapboxNavigation/EndOfRouteViewController.swift

```diff
@@ -69,137 +87,58 @@ class EndOfRouteViewController: UIViewController {
 
     // MARK: - Lifecycle Methods
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        self.clearInterface()
-        self.stars.didChangeRating = { [weak self] new in self?.rating = new }
-        self.setPlaceholderText()
-        self.styleCommentView()
-        self.commentViewContainer.alpha = 0.0 // setting initial hidden state
+    override func loadView() {
+        self.view = EndOfRouteContentView.forAutoLayout()
+        view.backgroundColor = .white
+        self.view.addSubview(self.column)
+        self.activateLayoutConstraints()
     }
     
     override func viewWillAppear(_ animated: Bool) {
-        super.viewWillDisappear(animated)
-        view.roundCorners([.topLeft, .topRight])
```

Comment:
After converting to a programmatic view, the view was not visible unless I commented out `view.roundCorners` here.

`view.roundCorners` depends on `view.bounds`, which are not necessarily at their final state in `viewWillAppear`, so I've moved it to viewDidLayoutSubiews, which guarantees the views bounds are set. 

Also this means we'll correctly re-compute the path if the bounds change (e.g. on device rotation).